src/plugin/localeData/index.js

```diff
@@ -12,6 +12,7 @@ export default (o, c, dayjs) => { // locale needed later
   }
   const getDayjsLocaleObject = () => dayjs.Ls[dayjs.locale()]
   const localeData = function () {
+    const t = format => format.replace(/(\[[^\]]+])|(MMMM|MM|DD|dddd)/g, (_, a, b) => a || b.slice(1))
```

Comment:
can we export this method from localizedFormat plugin for better code management?

Reply:
utils.js? what should I call it? makeLocalSmall again? :D

Reply:
No, what I think is simply export this function from  localizedFormat.js. This should not affect the plugin to work but also simply the logic here.